src/config/dynamic-theme-fixes.config

```diff
@@ -585,6 +585,27 @@ body[style] {
 
 ================================
 
+bestbuy.ca
+
+INVERT
+.centerContainer
+
+CSS
+[class^="facetName"], [class^="title"], [class^="sortLabel"], [class^="headerText"], [class^="subTitle"] {
+    filter:invert(100%) hue-rotate(180deg) contrast(90%) !important;
+}
+.backgroundContainer_ryHAa {
```

Comment:
In my testing, if I did not specify this exact selector, it did not work.

`when bestbuy pushes a CSS update it might change` Yes, that is true. As it is for all custom CSS in this config file.

Reply:
In my testing, if I did not specify this exact selector, it did not work.

```CSS
div[class^="backgroundContainer"]
```

Reply:
I'm not sure why that is as in my testing, if I use the selector as you have written it, the object disappears. No matter, I am changing this value in my pull request.

Reply:
Weird, what if you type the console 
```javascript
$('div[class^="backgroundContainer"]')
```
Does it still show up as a element or returns undefined?